pkg/registry/scripts/main.js

```diff
@@ -93,11 +99,21 @@
             /* Curtains related logic */
             function connect() {
                 $scope.curtains = { };
-                discover().then(function() {
+                discover().then(function(options) {
                     $scope.curtains = null;
+
+                    /* See if we have a bearer token to use */
+                    var authorization;
+                    if (options.headers) {
+                        authorization = options.headers['Authorization'] || "";
+                        $scope.config.registry.password = authorization.split("earer ")[1];
```

Comment:
True, I was worried about capitalization. But I guess I should just handle that correctly.